tests/DbTestCase.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+use October\Rain\Database\Model as ActiveRecord;
```

Comment:
Why name this ActiveRecord? Why not just use it as it is?

Reply:
don't know, I used what was in PluginTestCase...

Reply:
I would just remove it, it's not used like that anywhere in the actual code base.

Reply:
done!